lib/ansible/content.rb

```diff
@@ -0,0 +1,17 @@
+module Ansible
+  class Content
+    attr_accessor :path
+
+    def initialize(path)
+      @path = path
+    end
+
+    def fetch_galaxy_roles
+      roles_path = path.join('roles')
+      role_file  = path.join('requirements.yml')
```

Comment:
Should we check for both .yml and .yaml?

Reply:
This is all pulled directly from what the provider plugin stuff was doing. I can enhance it though.

Reply:
Would also have to change https://github.com/ManageIQ/manageiq/blob/master/lib/vmdb/plugins.rb#L46-L55 for this ...

It looks like this is the only caller though so I'm going to change the above method to look like the others then looks for both files here rather than assuming the directories are filtered out by the presence of a requirements file beforehand.